cpp/dolfinx/mesh/graphbuild.cpp

```diff
@@ -276,98 +285,158 @@ graph::AdjacencyList<std::int64_t> compute_nonlocal_dual_graph(
   MPI_Type_free(&compound_type);
   MPI_Comm_free(&comm_po_post);
 
+  // TODO: update
```

Comment:
Is this a future todo or a  current todo? As the code ha sbeen altered below, it's hard to tell if this should still be there.